src/main/java/seedu/java/util/Storage.java

```diff
@@ -0,0 +1,114 @@
+package seedu.java.util;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Loads and saves the taskList in .txt format.
+ * load()
+ * save()
+ */
+public class Storage {
+    String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads up a txt file and converts it to an Arraylist of Task.
+     * @return ArrayList of Tasks
+     * @throws FileNotFoundException means your text file is missing/corrupted
+     */
+    public ArrayList<Task> load() throws FileNotFoundException {
+        ArrayList<Task> arr = new ArrayList<>();
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        while (s.hasNextLine()) {
+            String line = s.nextLine();
+            String [] taskArr = line.split("/");
```

Comment:
I think you could remove the extra space between `String` and `[]`, just to adhere to the coding style